.gitignore

```diff
@@ -35,6 +35,7 @@ speed-measure-plugin*.json
 .history/*
 
 # misc
+/.angular/cache
```

Comment:
What's the purpose of this?

Reply:
From version 13.0.0 there will be a .angular folder generated in the root which is ignored by git containing a cache folder that caches builds.